### ISSUE TITLE
setRadioTick error

### ISSUE THREAD

**Issue by cristianmarian95** [jarvisteach/appJar#534]
When i try to use this ```self.app.setRadioTick(title, tick=True)``` i get  this error:
```python 
File "C:\Python\lib\site-packages\appJar\appjar.py", line 7471, in setRadioTick
    v.config(indicatoron=1)
  File "C:\Python\lib\tkinter\__init__.py", line 1482, in configure
    return self._configure('configure', cnf, kw)
  File "C:\Python\lib\tkinter\__init__.py", line 1473, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: unknown option "-indicatoron"
```

**Comment by jarvisteach** [comment]
I'm not seeing that problem...

Any chance you could provide:
* your platform/python/appJar versions
* and a sample of code that breaks

**Comment by jarvisteach** [comment]
HI @cristianmarian95 - are you using appJar in `ttk` mode?

**Comment by jarvisteach** [comment]
OK, I've looked into this a bit more.

Changing radio buttons is definitely not supported in ttk, or in fact on my test machine (a Mac), so I've introduced a warning when users try, and updated the docs to warn people.

I've also deprecated `setRadioTick()` as it doesn't actually turn radio buttons into tick boxes, instead turning them into big rectangular buttons. There's now a new function: `setRadioSquare` which will do the job. Again, docs updated accordingly.
